lib/github_advisory_sync.rb

```diff
@@ -308,7 +308,7 @@ def to_h
         "ghsa"        => ghsa_id[5..],
         "url"         => external_reference,
         "title"       => advisory["summary"],
-        "description" => advisory["description"],
+        "description" => advisory["description"].gsub(/description: (.*)/, 'description: |\n\1'),
```

Comment:
That `gsub` call is probably not going to do anything. The advisory metadata is pulled via GitHub's GraphQL end-point via `retrieve_all_rubygem_vulnerabilities`, so it won't contain the YAML `description: ...` key, it will just contain the description text.